src/plugins/vitest/index.ts

```diff
@@ -14,13 +14,13 @@ export const ENABLERS = ['vitest'];
 
 export const isEnabled: IsPluginEnabledCallback = ({ dependencies }) => hasDependency(dependencies, ENABLERS);
 
-export const CONFIG_FILE_PATTERNS = ['vitest.config.ts'];
+export const CONFIG_FILE_PATTERNS = ['vitest.config.ts', 'vitest.{workspace,projects}.{ts,js,json}'];
 
 // `TEST_FILE_PATTERNS` in src/constants.ts are already included by default
 export const ENTRY_FILE_PATTERNS = [];
 
 export const findVitestDeps = (config: VitestConfig) => {
-  if (!config || !config.test) return [];
+  if (!config || Array.isArray(config) || !config.test) return [];
```

Comment:
If this means to ignore what's exported from the new files than it would be better to bail out as soon as we know `configFilePath`, perhaps even without loading the file at all?

Reply:
In the end I changed my implementation to stick to what's possible with the workspace config.